src/main/java/mat/client/measure/ManageMeasureSearchModel.java

```diff
@@ -523,6 +525,24 @@ public void setVersion(String version) {
             this.version = version;
         }
 
+        /**
+         * Gets the isValidatable.
+         *
+         * @return the isValidatable
+         */
+        public boolean isValidatable() {
+            return this.isValidatable;
+        }
+
+        /**
+         * Sets the isValidatable.
+         *
+         * @param isValidatable the new isValidatable
```

Comment:
I think you missed equals/hashCode methods. I don't know why those are needed in the MAT App.. but their implementation is based on the fields/flags of the object.